app/components/UI/Notification/Row/Icon.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { View, Image } from 'react-native';
+import BadgeWrapper from '../../../../component-library/components/Badges/BadgeWrapper';
+import Badge, {
+  BadgeVariant,
+} from '../../../../component-library/components/Badges/Badge';
+import { BOTTOM_BADGEWRAPPER_BADGEPOSITION } from '../../../../component-library/components/Badges/BadgeWrapper/BadgeWrapper.constants';
+import { TRIGGER_TYPES } from '../../../../util/notifications';
+import { CommonSelectorsIDs } from '../../../../../e2e/selectors/Common.selectors';
+import NetworkMainAssetLogo from '../../NetworkMainAssetLogo';
+import { IconName } from '../../../../component-library/components/Icons/Icon';
+
+import RemoteImage from '../../../../components/Base/RemoteImage';
+import { ETHEREUM_LOGO } from '../../../../constants/urls';
+/* eslint-disable import/no-commonjs */
+const metamask_fox = require('../../../../images/fox.png'); // eslint-disable-line
+interface NotificationIconProps {
+  notificationType: TRIGGER_TYPES;
+  styles: any;
+  badgeIcon?: IconName;
+  imageUrl?: string;
+}
+function NotificationIcon({
+  notificationType,
+  styles,
+  badgeIcon,
+  imageUrl,
+}: NotificationIconProps) {
+  const customStyles = () => {
+    if (notificationType.toLowerCase().includes('erc721' || 'erc1155')) {
```

Comment:
As indicated by Sonar, this can't work: in JS, the || operator always return the first true item. And because items here are non empty strings, they are always considered as true. So the first one 'erc721' is always returned and the second one 'erc1155' is never considered in the includes. One way to fix that would be to match against `some` on an array (at least one element in the array passes) of notification types like the following:
```suggestion
    const types = ['erc721', 'erc1155'];
    if (types.some(type => notificationType.toLowerCase().includes(type))) {
```
This is not the only way of course. Feel free to adjust.

Reply:
Good catch. Fixed.